packages/vats/test/test-provisionPool.js

```diff
@@ -335,11 +335,12 @@ const makeWalletFactoryKitForAddresses = async addresses => {
 
       let isNew = !done.has(addr);
       if (isNew) {
-        const isRevive =
-          walletReviver && (await E(walletReviver).ackWallet(addr));
-        if (isRevive) {
-          isNew = false;
-        } else {
+        if (walletReviver) {
+          isNew = await E(walletReviver)
+            .ackWallet(addr, wallet)
+            .then(([_wallet, actualIsNew]) => actualIsNew);
+        }
+        if (isNew) {
```

Comment:
I wish we had more consistent terminology around the shop. Anything that's observable from a test is by definition not a refactor, the way I learned the term.

Reply:
Nit.
I would modify that slightly: anything that's observable by clients of the package...

The difference is that sometimes we arrange for our own tests to peek at our internals in a way that normal clients cannot.

Aside from the nit, I like your definition.